velox/exec/RowContainer.cpp

```diff
@@ -356,6 +362,16 @@ int32_t RowContainer::findRows(folly::Range<char**> rows, char** result) {
   return numRows;
 }
 
+void RowContainer::appendNextRow(char* existing, char* nextRow) {
```

Comment:
The naming of the parameters is a bit strange. Both rows exist, right? Perhaps, rename existing to 'row' or 'current'.